src/screens/TopicMemberScreen/index.js

```diff
@@ -88,7 +88,7 @@ const TopicMemberScreen = () => {
     return () => {
       scrollY.removeListener(listener);
     };
-  }, [scrollY]);
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependency array for the `useEffect` hook is empty, which means the effect will only run once after the initial render. If you want to clean up the event listener when `scrollY` changes, you should include `scrollY` in the dependency array.
```diff
-  }, []);
+  }, [scrollY]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->